diff -uNrp sakura-3.1.4.orig/sakura.desktop sakura-3.1.4/sakura.desktop
--- sakura-3.1.4.orig/sakura.desktop	2014-07-26 13:15:31.276061656 +1000
+++ sakura-3.1.4/sakura.desktop	2014-07-26 13:15:53.203597019 +1000
@@ -42,7 +42,7 @@ Comment[zh_CN]=使用命令行
 Comment[zh_TW]=使用指令列
 TryExec=sakura
 Exec=sakura
-Icon=terminal-tango
+Icon=utilities-terminal
 Type=Application
 Categories=GTK;Utility;TerminalEmulator;System;
 StartupNotify=true
